website/docs/deployment.mdx

```diff
@@ -389,11 +391,21 @@ jobs:
         with:
           node-version: 14.x
           cache: yarn
-      - name: Build website
-        working-directory: website
-        run: |
-          yarn install --frozen-lockfile
-          yarn build
+
+      - name: Get yarn cache
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-prescrypto-docs-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-prescrypto-docs-
```

Comment:
This is not needed either. All the things that need to be cached are in `node_modules`, and `node_modules` are automatically cached by setup-node.